### PR TITLE
New package: la-capitaine-icon-theme-0.6.1

### PR DESCRIPTION
**Pull request by LinArcX**
An icon pack designed to integrate with most desktop environments. The set of icons takes inspiration from the latest iterations of macOS and Google's Material Design:

https://github.com/keeferrourke/la-capitaine-icon-theme

### CODE REVIEW

File: srcpkgs/la-capitaine-icon-theme/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'la-capitaine-icon-theme'
+pkgname=la-capitaine-icon-theme
+version=0.6.1
+revision=1
+archs=noarch
+build_style=fetch
```
Comment (Chocimier) [inline srcpkgs/la-capitaine-icon-theme/template]: remove build_style=

File: srcpkgs/la-capitaine-icon-theme/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Icon pack designed to integrate with most desktop environments"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3"
```
Comment (Chocimier) [inline srcpkgs/la-capitaine-icon-theme/template]: GPL-3.0-only

File: srcpkgs/la-capitaine-icon-theme/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+homepage="https://github.com/keeferrourke/${pkgname}"
```
Comment (Chocimier) [inline srcpkgs/la-capitaine-icon-theme/template]: Replace all ${pkgname} with its value

File: srcpkgs/la-capitaine-icon-theme/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=9adb3123bf61464696400af43000b61de1825d2ce4b4ad4f21dd78091bd55e73
+
+do_install() {
+	tar xzf v${version}.tar.gz
```
Comment (Chocimier) [inline srcpkgs/la-capitaine-icon-theme/template]: remove, it is automatic when build_style is not set

File: srcpkgs/la-capitaine-icon-theme/template (hunk 3 above)
Comment (LinArcX) [reply srcpkgs/la-capitaine-icon-theme/template]: Why?

Comment (Chocimier) [reply srcpkgs/la-capitaine-icon-theme/template]: - Address can not be just copied out of template and pasted into browser.
- This is effectively constant so employing variable substitution is a waste.

File: srcpkgs/la-capitaine-icon-theme/template (hunk 4 above)
Comment (LinArcX) [reply srcpkgs/la-capitaine-icon-theme/template]: @Chocimier  If i remove this line, and remove `build_style=` line, it gives me these errors:
https://gist.github.com/LinArcX/034e69d3ca0063d30c183ac3e5e2e729

File: srcpkgs/la-capitaine-icon-theme/template (hunk 1 above)
Comment (LinArcX) [reply srcpkgs/la-capitaine-icon-theme/template]: Done!

File: srcpkgs/la-capitaine-icon-theme/template (hunk 2 above)
Comment (LinArcX) [reply srcpkgs/la-capitaine-icon-theme/template]: Done!

**Comment by Piraty** [comment]
As discussed a few times before (in the old, now gone, gh-repo and on IRC), Void prefers to not package more themes, icons, mouse cursors etc., especially if they are derivatives of other themes, possibly contain work of dubious licensing situation or are expected to not be of interest for most users.
The policy mostly was/is to provide themes that are used by the main desktop environments, and it is expected that such things can easily installed by users on their own.

**Pull request closed without merging**